test/common/common/BUILD

```diff
@@ -389,6 +390,9 @@ envoy_cc_test(
 envoy_cc_test(
     name = "version_test",
     srcs = ["version_test.cc"],
+    copts = envoy_select_boringssl(
+        ["-DENVOY_SSL_FIPS"],
+    ),
```

Comment:
Nit: This could have been written as `copts = envoy_select_boringssl(["-DENVOY_SSL_FIPS"]),`